docs/pages/es/installation/windows.md

```diff
@@ -0,0 +1,140 @@
+Se recomienda actualizar a <a href="#wsl2">WSL2</a> ya que WSL1 no soporta la aplicación Docker Desktop para Windows. En su lugar, solo soporta la versión obsoleta, [Docker Toolbox](https://docs.docker.com/toolbox/toolbox_install_windows/).
+
+Si aun desea continuar, siga las instrucciones para <button onclick="HideToggleFunction()"><b>WSL1</b></button>
+
+<div id="hiddendiv">
+<p>
+1. La versión por defecto de WSL está establecida en WSL1 por defecto. Puedes ir al siguiente link para <a href="https://docs.microsoft.com/en-us/windows/wsl/install-win10#install-your-linux-distribution-of-choice">instalar la distro</a> de tu elección. <br /><br />
+
+2. <b><a href="https://docs.docker.com/toolbox/toolbox_install_windows/">Docker Toolbox</a></b> <br />
+
+<b>Advertencia</b>: Docker Toolbox es una versión obsoleta. Se recomienda actualizar el sistema e instalar la aplicación Docker Desktop con WSL2. <br/><br />
+
+Docker Toolbox utiliza características específicas del kernel de Linux, y no puede funcionar de forma nativa en Windows. En su lugar, crea y utiliza una pequeña VM de Linux en tu máquina junto con <a href="https://docs.docker.com/machine/overview/"><code>docker-machine</code> </a>, y utiliza VirtualBox para ejecutar Docker. <br />
+    <ul>
+       <li>  Valla a <a href="https://github.com/docker/toolbox/releases">Toolbox Releases</a> y descargue la última versión del archivo <code>.exe</code></li>
+       <li> Siga estas <a href="https://docs.docker.com/toolbox/toolbox_install_windows/#step-2-install-docker-toolbox">instrucciones</a> para configurar con éxito la aplicación Docker Toolbox. </li>
+    </ul>
+
+</p>
+</div>
+
+
+#### 2. <b>[Instalar la nueva distro](https://docs.microsoft.com/en-us/windows/wsl/install-win10#install-your-linux-distribution-of-choice)</b>
+En este tutorial, [Ubuntu 18.04](https://www.microsoft.com/en-us/p/ubuntu-1804-lts/9n9tngvndl3q?activetab=pivot:overviewtab) será la distro utilizada. Siéntase libre de usar cualquier distro a su elección.
+
+
+#### 3. <b>habilitar Docker</b>
+
+La aplicación Docker Desktop para Windows incluye un completo conjunto de herramientas, incluyendo Docker Engine, el cliente Docker CLI, Docker Compose, Notary, Kubernetes, y un Credential Helper.
+
+<table id="compatibility-table">
+  <tr>
+    <th id="model">Versión Windows 10</th>
+    <th id="model">Docker Desktop</th>
+  </tr>
+  <tr>
+    <td>Pro/Education/Enterprise</td>
+    <td><a href="https://docs.docker.com/docker-for-windows/install/">Docker Desktop para Windows Pro</a></td>
+  </tr>
+  <tr>
+    <td>Home</td>
+    <td><a href="https://docs.docker.com/docker-for-windows/install-windows-home/">Docker Desktop para Windows Home</a></td>
+  </tr>
+</table>
+
+#### 4. <a name="step4"> <b>Instalar un cluster Kubernetes</b></a>
+
+Una vez que Docker esté instalado, el siguiente paso será instalar un cluster Kubernetes. En este como-hacerlo, se usará [K3d](https://github.com/rancher/k3d) ya que sólo depende de Docker.
```

Comment:
este como-hacerlo ?

Reply:
Acepto sugerencias o simplemente ese no se traduce, la palabra es how-to

Reply:
Sugiere poner en este procedimiento

Reply:
@jeojhx puedes usar: en esta guía en vez de en este how-to.

Reply:
Excelente... Realizo los cambios...